src/mono/wasm/wasm.proj

```diff
@@ -110,4 +108,109 @@
           SkipUnchangedFiles="true" />
   </Target>
 
+  <UsingTask TaskName="GenerateWasmVersionFile"
+    AssemblyFile="$(WasmBuildTasksAssemblyPath)"/>
+  <Target Name="BuildWinWasmRuntimes"
```

Comment:
Can this target replace the `make` invocation in `BuildWasmRuntimes`? I mean, are they two effectively the same?

Reply:
Yes